deployments/cee/image/environment.yml

```diff
@@ -19,3 +22,7 @@ dependencies:
 # For desktop environment
   - jupyter-desktop-server==0.1.3
   - -r infra-requirements.txt
+  
+  #needed for math functions
+  - scipy
```

Comment:
Can you add these under where you added `matplotlib` as well, so these all can come from `conda-forge`? 

In general, I think everything should come from conda-forge, unless the version there is too old or doesn't exist. mpld3 exists there too, for example (https://anaconda.org/conda-forge/mpld3)